modules/uid2IdSystem_shared.js

```diff
@@ -670,13 +752,9 @@ export function Uid2GetId(config, prebidStorageManager, _logInfo, _logWarn) {
     originalToken: suppliedToken ?? storedTokens?.originalToken,
     latestToken: newestAvailableToken,
   };
-  if (cstgIdentity) tokens.originalIdentity = storedTokens?.originalIdentity
+  if (FEATURES.UID2_CSTG && isCstgEnabled) {
```

Comment:
Do we need to check `FEATURES.UID2_CSTG` here?

Reply:
we could remove it, with `FEATURES.UID2_CSTG` line 709-726 will be removed from build, just reduce some bytes